Automattic/jetpack
# CPT: fix PHPCS linting errors in CPT module files, part 2

See #21963

#### Changes proposed in this Pull Request:

* Fix PHPCS errors in Testimonials and in the main module file.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Go to Jetpack > Settings > Writing and enable Testimonials
* Go to the new Testimonials menu and create a few Testimonials.
* Go to Posts > Add New, and add a new shortcode as explained [here](https://wordpress.com/support/testimonials/#testimonial-shortcode).
* See how it goes.
